fcrepo-service-ldcache-file/src/main/resources/OSGI-INF/blueprint/ldcache-service.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The pattern you have established with the other modules is to name this file: `blueprint.xml`.
Is there any technical reason for the `blueprint.xml` naming convention? or for changing it for this module?